project/Merging.scala

```diff
@@ -0,0 +1,57 @@
+import sbt._
```

Comment:
It's used for de-duping during sbt assembly. It's just a bunch of crapola that was cluttering up build.sbt